### PR TITLE
Add live views to new webui

### PR DESCRIPTION
**Pull request by NickM-27**
For now just adding webrtc and debug because MSE and JSMPEG rely on javascript which is making things less simple. Persistence is supported and also fixed an issue on mobile with screen width

### CODE REVIEW

**Review by hawkeye217 — COMMENTED** [review]
So I had originally created a "context" directory for hooks, but "hooks" is probably more appropriate. Maybe you could move `use-persistence.tsx` into `hooks` and then instead of `index.ts`, make a `resize-observer.tsx` or something if we want to keep separate hooks in their own files.

**Comment by NickM-27** [comment]
@hawkeye217 good idea, done 👍

File: web-new/src/components/player/LivePlayer.tsx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+import { CameraConfig } from "@/types/frigateConfig";
+import AutoUpdatingCameraImage from "../camera/AutoUpdatingCameraImage";
+import ActivityIndicator from "../ui/activity-indicator";
+import { Button } from "../ui/button";
+import { LuSettings } from "react-icons/lu";
+import { useCallback, useMemo, useState } from "react";
+import { Card, CardContent, CardHeader, CardTitle } from "../ui/card";
+import { Switch } from "../ui/switch";
+import { Label } from "../ui/label";
+import { usePersistence } from "@/context/use-persistence";
```
Comment (hunterjm) [inline web-new/src/components/player/LivePlayer.tsx:11]: This context is missing from this PR and doesn't exist on the 0.14 branch.

Comment (hunterjm) [reply web-new/src/components/player/LivePlayer.tsx:11]: Looks like it was moved to `hooks` and missed here.

Comment (NickM-27) [reply web-new/src/components/player/LivePlayer.tsx:11]: Yeah missed that one, fixed in https://github.com/blakeblackshear/frigate/pull/8981

**Pull request merged**